justfile

```diff
@@ -10,6 +10,9 @@ build:
 test: build
   cargo test
 
+test-count:
```

Comment:
If you add a comment above it will show in `just -l` https://raw.githubusercontent.com/casey/just/master/screenshot.png

Reply:
nice, thanks. done